### DISCUSSION TITLE
blog/building-random-gif-generator-with-fets-and-giphy

### DISCUSSION THREAD

**Discussion by giscus** [the-guild-org/the-guild-website#1611] [Docs Discussions]
# blog/building-random-gif-generator-with-fets-and-giphy

Building a Random Gif Generator using the Giphy API and feTS, a tool designed to simplify REST API development.

https://the-guild.dev/blog/building-random-gif-generator-with-fets-and-giphy

**Comment by predragnikolic** [comment]
feTS is one of the exciting libraries of 2024!
Hope that more people discover it
